products/apache-apisix.md

```diff
@@ -13,17 +13,23 @@ identifiers:
 
 auto:
   methods:
-  -   git: https://github.com/apache/apisix.git
+  -   github_releases: 'apache/apisix'
```

Comment:
```suggestion
  -   github_releases: apache/apisix
```

:lipstick: Quotes  are not necessary

Reply:
👍

Reply:
gonna remove quotes across the board, https://github.com/endoflife-date/endoflife.date/pull/5694